mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -3635,6 +3636,63 @@ static uint64_t getTeamsReductionDataSize(mlir::omp::TeamsOp &teamsOp) {
   return getReductionDataSize<mlir::omp::TeamsOp>(teamsOp);
 }
 
+static void
```

Comment:
Could you please add a comment here? The name is self-explanatory so a smallish comment should be good enough